src/leap/bitmask/services/mail/smtpbootstrapper.py

```diff
@@ -99,9 +102,21 @@ def _download_config_and_cert(self):
                 check_and_fix_urw_only(client_cert_path)
                 return
 
-            download_client_cert(self._provider_config,
-                                 client_cert_path,
-                                 self._session)
+            try:
+                download_client_cert(self._provider_config,
+                                     client_cert_path,
+                                     self._session, kind="smtp")
+            except HTTPError:
```

Comment:
shouldn't we catch specifically `403`? can we? are there any other error cases that we may supress like this?